README.md

```diff
@@ -43,6 +43,20 @@ To run a specific script, run
 ```
 npx hardhat run <PATH_TO_SCRIPT>
 ```
+
+## Code Coverage ##
+
+To generate code coverage metrics via instrumented runs of the testing suite, use:
+
+```sh
```

Comment:
is this `sh` meant to be here? If so thats cool, is it identifying the code snippet?

Reply:
> is this sh meant to be here? If so thats cool, is it identifying the code snippet?

@raymogg [Yes](https://github.github.com/gfm/#example-112), [it](https://github.github.com/gfm/#info-string) is!